metadata-ingestion/src/datahub/ingestion/source/neo4j/neo4j_source.py

```diff
@@ -40,35 +33,26 @@
     StatefulIngestionReport,
     StatefulIngestionSourceBase,
 )
-from datahub.metadata.com.linkedin.pegasus2avro.schema import SchemaFieldDataType
-from datahub.metadata.schema_classes import (
-    AuditStampClass,
-    BooleanTypeClass,
-    DatasetPropertiesClass,
-    DateTypeClass,
-    NullTypeClass,
-    NumberTypeClass,
-    OtherSchemaClass,
-    SchemaFieldClass,
-    SchemaMetadataClass,
-    StringTypeClass,
-    SubTypesClass,
-    UnionTypeClass,
-)
+from datahub.sdk.dataset import Dataset
 
 log = logging.getLogger(__name__)
 logging.basicConfig(level=logging.INFO)
 
-_type_mapping: Dict[Union[Type, str], Type] = {
-    "list": UnionTypeClass,
-    "boolean": BooleanTypeClass,
-    "integer": NumberTypeClass,
-    "local_date_time": DateTypeClass,
-    "float": NumberTypeClass,
-    "string": StringTypeClass,
-    "date": DateTypeClass,
-    "node": StringTypeClass,
-    "relationship": StringTypeClass,
+# Neo4j object types
+_NODE = "node"
+_RELATIONSHIP = "relationship"
+
+# Map Neo4j types to DataHub types
+_STRING_TYPE_MAPPING: Dict[str, str] = {
```

Comment:
Don't we have some shared place about the DataHub types?
These strings scare me and are easy to make.